ios/UIViewController+RNScreens.m

```diff
@@ -0,0 +1,52 @@
+#import "UIViewController+RNScreens.h"
+#import "RNSScreenContainer.h"
+#import "RNSScreenStack.h"
+#import "RNSScreen.h"
+
+#import <objc/runtime.h>
+
+
+@implementation UIViewController (RNScreens)
+
+- (UIViewController *)reactNativeScreensChildViewControllerForStatusBarStyle
+{
+  UIViewController* lastViewController = [[self childViewControllers] lastObject];
+  if ([lastViewController isKindOfClass:[RNScreensNavigationController class]] || [lastViewController isKindOfClass:[RNScreensViewController class]] || [lastViewController isKindOfClass:[RNSScreen class]]) {
```

Comment:
maybe we can define a protocol that all RNScreen navigators conform to in order to avoid all these checks.